### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by goldingn** [zoonproject/zoon#106]
#### TL;DR: I think we should explicitly state the needed packages in the readme, PR on its way.

I've having some trouble with the current [stable version installation instructions](https://github.com/zoonproject/zoon#to-install-the-stableish-version-directly-from-r) on my mac:

> and on OSX or Linux:
```r
# install the Windows binary of the latest release
install.packages('https://github.com/zoonproject/zoon/releases/download/0.3.1/zoon_0.3.1.tar.gz',
               repos = NULL, method = 'libcurl')

># install dependencies
install.packages(pkgDepends('zoon')$Depends)

># install imports
install.packages(unlist(strsplit(gsub(',', '', installed.packages(fields = c('imports'))['zoon','Imports']), '\n'))[-1])

># load zoon
library(zoon)
```

To start with I get this:

```r
# install dependencies
install.packages(pkgDepends('zoon')$Depends)
```
`Error in install.packages : could not find function "pkgDepends"`

Which is easily solved by:

```r
# install dependencies
install.packages(tools::pkgDepends('zoon')$Depends)
```

Then I get:

```r
# install imports
install.packages(unlist(strsplit(gsub(',', '', installed.packages(fields = c('imports'))['zoon','Imports']), '\n'))[-1])
```
`Error in install.packages : no packages were specified`

which fails because:
```r
unlist(strsplit(gsub(',', '', installed.packages(fields = c('imports'))['zoon','Imports']), '\n'))
```
returns the string:
`"assertthat rlist RCurl httr httpuv dismo"`
rather than a vector, and because the `[-1]` bit causes it to return a vector of length 1.

The following code seems to work:
```r
install.packages(unlist(strsplit(gsub(', ', '\n', installed.packages(fields = c('imports'))['zoon','Imports']), '\n')))
```

as
```r
unlist(strsplit(gsub(', ', '\n', installed.packages(fields = c('imports'))['zoon','Imports']), '\n'))
```
returns
`[1] "assertthat" "rlist"      "RCurl"      "httr"       "httpuv"    
[6] "dismo"`

In other words, The OSX bit should read:

```r
# install the tarball of the latest release
install.packages('https://github.com/zoonproject/zoon/releases/download/0.3.1/zoon_0.3.1.tar.gz',
                 repos = NULL, method = 'libcurl')

# install dependencies
install.packages(tools::pkgDepends('zoon')$Depends)

# install imports
install.packages(unlist(strsplit(gsub(', ', '\n', installed.packages(fields = c('imports'))['zoon','Imports']), '\n')))

# load zoon
library(zoon)
```

On my linux box (using RStudioServer) this doesn't work, I get:
```r
install.packages('https://github.com/zoonproject/zoon/releases/download/0.3.1/zoon_0.3.1.tar.gz',
                 repos = NULL, method = 'libcurl')
```
```
ERROR: dependencies ‘assertthat’, ‘rlist’, ‘httpuv’ are not available for package ‘zoon’
* removing ‘/home/lina1864/R/x86_64-pc-linux-gnu-library/3.2/zoon’
Warning in install.packages :
  installation of package ‘/tmp/RtmpuGUT0X/downloaded_packages/zoon_0.3.1.tar.gz’ had non-zero exit status
```

```r
install.packages(pkgDepends('zoon')$Depends)
```
`Error in install.packages : could not find function "pkgDepends" `

I guess this is just because I've not installed these things on that machine before, so this approach isn't likely to work.

Instead, I suggest we change installation instructions for both Windows and OSX/linux to explicitly state the packages needed I.e.:

```r
# install all of the dependencies
install.packages(c("methods", "raster", "sp (>= 1.0-13)", "assertthat", "rlist", "RCurl", "httr", "httpuv", "dismo"))

# install the tarball of the latest release
install.packages('https://github.com/zoonproject/zoon/releases/download/0.3.1/zoon_0.3.1.tar.gz',
                 repos = NULL, method = 'libcurl')
```

This works just fine, but will need editing for future versions. Will submit a PR shortly.

(this is also why #103 may be a good idea...)

**Comment by AugustT** [comment]
I had to Google TL;DR ... it was too late

**Comment by goldingn** [comment]
I tried :)
